sql/core/src/test/scala/org/apache/spark/sql/SQLQuerySuite.scala

```diff
@@ -2773,4 +2773,22 @@ class SQLQuerySuite extends QueryTest with SharedSQLContext {
       }
     }
   }
+
+  Seq("orc", "parquet", "csv", "json").foreach { format =>
+    test(s"Write and read back unicode schema - $format") {
```

Comment:
instead of keeping adding test cases in `SQLQuerySuite`, shall we create a dedicate test suite for file based data source now?

Reply:
+1. That's a best idea. I'll update like that.